maintenance/importConfig.ts

```diff
@@ -134,35 +225,18 @@ async function downloadOzwConfig(): Promise<string> {
 }
 
 /** Extract `config` folder from ozw archive in `tmpDir` */
-function extractConfigFromTar(): Promise<void> {
-	return new Promise((resolve, reject) => {
-		console.log("extracting config folder from ozw archive...");
-		child.exec(
-			`tar -xzf ${ozwTarName} open-zwave-master/config  --strip-components=1`,
-			{ cwd: tmpDir },
-			function (error) {
-				if (error) {
-					reject(error);
-				} else {
-					resolve();
-				}
-			},
-		);
-	});
+async function extractConfigFromTar(): Promise<void> {
+	console.log("extracting config folder from ozw archive...");
+	await execPromise(
+		`tar -xzf ${ozwTarName} open-zwave-master/config  --strip-components=1`,
+		{ cwd: tmpDir },
+	);
```

Comment:
cool. I think promisify-child-process did some small extras, but the built-ins should be enough for our purpose.